internal/pkg/service/common/etcdop/key.go

```diff
@@ -106,7 +106,7 @@ func (v Key) PutIfNotExists(val string, opts ...etcd.OpOption) op.BoolOp {
 	)
 }
 
-func (v KeyT[T]) Get(opts ...etcd.OpOption) op.GetOneTOp[T] {
+func (v KeyT[T]) Get(opts ...etcd.OpOption) op.ForType[*op.KeyValueT[T]] {
```

Comment:
V tomto commite som upravil operacie tak, aby sa dali spajat jednoducho do transakcii.
Implementuju spolocny interface `Op` (nizsie).

Okrem toho, som prisiel na to, ako to zapisat genericky.
Ma to jednu nevyhodu, ze pre genericke typy sa neda nefinovat alias:
```
type GetManyOp = op.ForType[*op.KeyValueT[T]]
```